source/features/batch-mark-files-as-viewed.tsx

```diff
@@ -38,10 +41,27 @@ function batchToggle(event: delegate.Event<MouseEvent, HTMLFormElement>): void {
 	}
 }
 
+function markAsViewedSelector(target: HTMLElement): string {
+	const checked = (target as HTMLInputElement).checked ? ':not([checked])' : '[checked]';
+	return '.js-reviewed-checkbox' + checked;
+}
+
+const markAsViewed = clickAll(markAsViewedSelector);
+
+async function onAltClick(event: delegate.Event<MouseEvent, HTMLElement>): Promise<void> {
+	if (event.altKey && event.isTrusted) {
+		new Toast().show();
+		await delay(30); // Without this, the Toast doesn't appear in time
+		markAsViewed(event);
+		new Toast().done();
```

Comment:
It's not a new toast every time. It should be something like
```suggestion
		const toast = new Toast();
		await delay(30); // Without this, the Toast doesn't appear in time
		markAsViewed(event);
		toast.off();
```